spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/SamplingZipkinSpanIterator.java

```diff
@@ -38,6 +41,23 @@
  * @since 1.0.0
  */
 final class SamplingZipkinSpanIterator implements Iterator<zipkin.Span> {
+	private static final List<String> ZIPKIN_CLIENT_ANNOTATIONS = Arrays.asList(
```

Comment:
ps I understand why you did this, just that the logic in zipkin is a little overly cautious :P